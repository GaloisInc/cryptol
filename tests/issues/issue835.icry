:m Float
:t number`{1, Float _ _}
