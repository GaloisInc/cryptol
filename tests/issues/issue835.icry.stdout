Loading module Cryptol
Loading module Cryptol
Loading module Float
number`{1, Float _ _} :
  {n, m} (ValidFloat n m, Literal 1 (Float n m)) => Float n m
